Project-business-site - Maidelin Rubio

rubmaigut opened the pull request.

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+      <!-- Navigation Bar (Menu  -->
+      <div class="menu-hamburguesa">
+        <a class="button" href="http://127.0.0.1:5500/code/index.html#modal">
+          <img src="./Project 3_files/icon-mobile.png"></a>
```
Let's remember that all files as well as folders should be named with lowercase letters - `project` instead of `Project`. Let's fix that ;)

rubmaigut replied:
Hi Max, I forgot to remove this file because it's no used in the project.

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+        <a class="button" href="http://127.0.0.1:5500/code/index.html#modal">
+          <img src="./Project 3_files/icon-mobile.png"></a>
+        <div id="modal" class="overlay">
+          <a class="cancel" href="http://127.0.0.1:5500/code/index.html#"> </a>
```
You anchor is empty, so there is no way of clicking it, we should add some text inside.
Moreover, `href` is not correct - I think you wanted write `href="index.html"`

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+      <!-- Navigation Bar (/Search  :) -->
+      <div class="search">
+        <input type="text" placeholder="Search..">
+        <button type="submit"> <i class="fa-search"></i>
```
Wrong indentations - it should rather be 

```
<button type="submit">
  <i class="fa-search"></i>
  <img />
</button>
```

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+    <!-- Hero Image / Video :) -->
+    <section class="hero-home">
+      <div class="hero-background">
+        <img src="./Project 3_files/Color-explosion.png">
```
capitalised folder name here as well

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+      <!-- Signup form (Personal data  -->
+      <fieldset>
+        <legend>You Account</legend>
+        <label for="fname">First Name</label><br>
```
We shouldn't use `<br />` tags, let's delete them

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+      <p>Lorem ipsum...</p>
+    </div>
+  </section>
+  <script src="./Project 3_files/script.js.descarga"></script>
```
script file should have extension `.js`, without anything else.
There is also another thing - I've noticed you use spanish words from time to time (in class name, or over here). We should avoid that - english language is like a latin of developers, we communicate between countries thanks to that. This rule should be applied to everything we write :)

EmOlsz commented on img/Project 3.html:
```diff
@@ -0,0 +1,196 @@
+  <script src="./Project 3_files/script.js.descarga"></script>
+  <!-- dont forget to link the javascript file here! -->
+<!-- Code injected by live-server -->
+<script type="text/javascript">
```
Hmm, I don't think you need this. What was the purpose if this code?

EmOlsz commented on img/Project 3_files/mobile-style.css:
```diff
@@ -0,0 +1,167 @@
+  background-image: url(/img/background.png);
+}
+
+fieldset{
```
Let's remember we should style by class names, not by selector - needs to be fixed

EmOlsz commented on index.html:
```diff
@@ -0,0 +1,227 @@
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="./styles/mobile-style.css">
```
I admire your attention to details, but one CSS file is completely enough, we don't have to defragmentize them :)

rubmaigut replied:
Hi Max, 

In this case, is better for me separated the CSS by feature or functionality so, I can have the code organized and this makes it easier for me to understand what I am doing. 

Thanks for your comments I will have it in mind the next time.

EmOlsz reviewed (changes requested):
Hey Maidelin!
Thank your for your pull request! First thing I wanted to say is that I'm not able to each your website, link is not working. Could you fix that?
Apart from that, in general code needs some fixes. I wrote you some comments, take a look at them and provide changes. I would also like to see your website, so remember to add link :)
Waiting for your reply!

rubmaigut commented:
> Hey Maidelin!
> Thank your for your pull request! First thing I wanted to say is that I'm not able to each your website, link is not working. Could you fix that?
> Apart from that, in general code needs some fixes. I wrote you some comments, take a look at them and provide changes. I would also like to see your website, so remember to add link :)
> Waiti

rubmaigut commented:
Hi @EmOlsz

I updated the project (readme/files) because I realized that the file that you were reviewing was the wrong one. I forgot to remove it because this was the file where I started to code but then I changed for the index.html that is in the made route of the folder.

This is the link to the live pages. 

https://woodmood-project.netlify.app

EmOlsz commented on index.html:
```diff
@@ -0,0 +1,227 @@
+        <input class="box" type="email" id="email" name="email" required><br>
+
+        <label class="label" for="pass">Phone Number</label><br>
+        <input class="box" type="tel" id="phone" name="phone" placeholder="123-456-7890"
```
`for` and `id` attributes doesn't match over here, let's fix that

EmOlsz commented on index.html:
```diff
@@ -0,0 +1,227 @@
+        <legend>Subject of your Consultation</legend>
+        <div class="themes-options">
+          <div class="option">
+            <input class="radio" type="radio" id="furniture" value="furniture">
```
When it comes to radio buttons, there is one thing that we should be super aware of : user should be allowed to select only one radio button at a time. In your case however, I was able to select all of them.
How can we fix that? We need to add `name` attribute to all of them, moreover, this attribute has to have same value for all radio buttons, like this :

```
<input type="radio" name="theme" value="furniture" />
<input type="radio" name="theme" value="floaring" />
<input type="radio" name="theme" value="illustration" />
<input type="radio" name="theme" value="mobile-art" />
```

rubmaigut replied:
Hey, @EmOlsz I've done the changes but after pushing them, I don't see my changes in the PR. Also, it says that the PR is closed. Maybe is because of that?

EmOlsz replied:
Hey, indeed, it's closed, you closed it 3 days ago :D If you will reopen it, then we should see the changes!
<img width="1049" alt="Screenshot 2021-02-10 at 11 53 41" src="https://user-images.githubusercontent.com/22129355/107506931-1d458700-6b9f-11eb-9bcb-eb71efd47e59.png">

rubmaigut replied:
HI @EmOlsz 

I didn't realize that was me who closed the PR. Now it's ready.

EmOlsz commented:
Hey Maidelin!
Thank you for your comments, I understand situation better now :)
I've added to extra comments (present above this message), take a look on them and provide changes.
After that, I'll approve project!

EmOlsz commented:
Thank you, project approved :)
